index.js

```diff
@@ -88,18 +88,18 @@ class NetworkResource extends Nanoresource {
     }
   }
 
-  announce (key) {
+  announce (key, { lookup = false } = {}) {
     if (!this.discovery) throw new Error('Bind before announcing')
     const localPort = this.tcp.address().port
-    return this.discovery.announce(key, { localPort })
+    return this.discovery.announce(key, { port: 0, localPort, lookup })
```

Comment:
was this default false before also?

Reply:
it wasn't set before - so default: `undefined` - which is also the default in discovery

so the practical outcome that it was default false before